phpunit.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd"
+        bootstrap="vendor/autoload.php"
+        colors="true"
+        beStrictAboutTestsThatDoNotTestAnything="true"
+        beStrictAboutOutputDuringTests="true"
+        beStrictAboutChangesToGlobalState="true"
+        failOnNotice="true"
+        failOnWarning="true"
+        displayDetailsOnTestsThatTriggerNotices="true"
+        displayDetailsOnTestsThatTriggerWarnings="true"
+        displayDetailsOnTestsThatTriggerErrors="true"
+        cacheDirectory=".cache/.phpunit.cache"
```

Comment:
Can we then just use ".cache/phpunit"? No need to make it hidden with "." in the ".cache" folder.

Reply:
I think ".phpunit.cache" is like the standard name of phpunit cache. Why should we rename it and confuse ppl?

Reply:
Which people care about a phpunit cache directory, but okay ...

Reply:
I actually had problems where I needed to delete it to get rid of problems in the guides several times...